src/types.ts

```diff
@@ -222,14 +296,37 @@ type TransactionAddress = {
   type: TransactionAddressType
 }
 
+type TransactionSumEvidence = {
+  currency?: string
+  amount: number
+}
+
 type PaymentValue = {
   amount: number
   currency?: string
 }
 
+export enum CouponType {
+  PercentOff = 'percent_off',
+  FixedValue = 'fixed_value'
+}
+
+type Coupon = {
+  id?: string
+  type?: CouponType
+  value?: number
+  maxDiscount?: number
+  name?: string
+}
+
 export enum PaymentMethodType {
   CreditCard = 'credit_card',
-  DebitCard = 'debit_card'
+  DebitCard = 'debit_card',
+  ApplePay = 'apple_pay',
+  GooglePay = 'google_pay',
+  NuPay = 'nu_pay',
+  Pix = 'pix',
+  MealVoucher = 'meal_voucher'
```

Comment:
Can you please add "account_balance" method as well? It's not on the doc yet, but it will be added soon